<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminSettingUI</class>
 <widget class="QWidget" name="AdminSettingUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminSettingUI</string>
  </property>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>361</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_roiHeight_L">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>最大运动行程</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_MaxMotionStroke">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>1441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 250);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>22</pointsize>
     <weight>12</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 100 22pt &quot;SimSun-ExtB&quot;;</string>
   </property>
   <property name="text">
    <string>PCB检测系统 - 系统设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>830</y>
     <width>112</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>830</y>
     <width>112</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>361</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_roiHeight_L_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>相机总数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_MaxCameraNum">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>1441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>361</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_roiHeight_L_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>每毫米像素数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_PixelsNumPerUnitLength">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>361</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="label_roiHeight_L_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>145</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>145</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>分图重叠率(宽)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ImageOverlappingRate_W">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>361</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_roiHeight_L_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>分图尺寸(pix)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_roiHeight_L_9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>宽</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ImageSize_W">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>3</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_roiHeight_L_10">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>21</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>21</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>高</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ImageSize_H">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>65</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_11">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>361</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="label_roiHeight_L_11">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>145</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>145</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>分图重叠率(高)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_12">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ImageOverlappingRate_H">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_12">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>361</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QLabel" name="label_roiHeight_L_12">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>145</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>145</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>单位脉冲数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_13">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_PulseNumInUnitTime">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
